NOMIS/nomis.go

```diff
@@ -109,13 +116,32 @@ func main() {
 		return
 	}
 
-	for index0, _ := range res.Structure.Keyfamilies.Keyfamily {
+	for index0 := range res.Structure.Keyfamilies.Keyfamily {
+		var annotations = res.Structure.Keyfamilies.Keyfamily[index0].Annotations.Annotation
 		censusEditionData := models.EditionUpdate{}
+		censusInstances := models.Version{}
 		mapData := models.Dataset{}
-		cenId := res.Structure.Keyfamilies.Keyfamily[index0].ID
-		mapData.Title = res.Structure.Keyfamilies.Keyfamily[index0].Name.Value
-		mapData.ID = cenId
+		var cenId string
+		for censusId := range annotations {
+			annoIndex := res.Structure.Keyfamilies.Keyfamily[index0].Annotations.Annotation[censusId]
+			switch annoIndex.Title {
+			case "Mnemonic":
+				ref := annoIndex.Text.(string)
+				cenParam := strings.Split(ref, "c2011")
```

Comment:
Not a blocker to the PR, but don't think it should have `param` in its name and cenParam (I assume is census parameter) is very generic, when reading the code I have to look to see how the variable is used to understand what it is. Ideally, it should be named in a way that from just reading the variable name I know what it is.